examples/list_of_lists/test/test.ts

```diff
@@ -0,0 +1,535 @@
+import { Principal } from '@dfinity/principal';
+import { run_tests, Test } from 'azle/test';
+// import { get_tests } from 'azle/examples/list_of_lists/test/tests';
+import { execSync } from 'child_process';
+import { createActor } from './dfx_generated/list_of_lists';
+import { deepEqual } from 'fast-equals';
+import { Opt } from 'azle';
+
+const list_of_lists_canister = createActor('rrkah-fqaaa-aaaaa-aaaaq-cai', {
+    agentOptions: {
+        host: 'http://127.0.0.1:8000'
+    }
+});
+
+function arrEqual<T>(a: DeepArray<T>, b: DeepArray<T>): boolean {
+    if (Array.isArray(a) && Array.isArray(b)) {
+        return (
+            a.length === b.length &&
+            a.every((value, index) => arrEqual(value, b[index]))
+        );
+    }
+    if (Array.isArray(a) || Array.isArray(b)) {
+        return false;
+    }
+    return (
+        a.length === b.length && a.every((value, index) => value === b[index])
+    );
+}
+
+type DeepArray<T> = arr | Array<arr | DeepArray<T>>;
+
+type arr =
+    | Int8Array
+    | Int16Array
+    | Int32Array
+    | BigInt64Array
+    | Uint8Array
+    | Uint16Array
+    | Uint32Array
+    | BigUint64Array;
+
+const tests: Test[] = [
```

Comment:
Can you put these tests in a sibling file called `tests.ts` and import them?

Reply:
This is how it is in azle and I will just get rid of the tests once we have it merged in azle